src/stan/lang/generator/get_typedef_var_type.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef STAN_LANG_GENERATOR_GET_TYPEDEF_VAR_TYPE_HPP
+#define STAN_LANG_GENERATOR_GET_TYPEDEF_VAR_TYPE_HPP
+
+#include <stan/lang/ast.hpp>
+#include <string>
+
+namespace stan {
+  namespace lang {
+    /**
+     * Return cpp type name or typedef used for bare_expr_type.
+     * NOTE: (26/02/2018)
```

Comment:
Fix this so it looks like real doc.  There shouldn't be stray notes like this in the actual doc that will be exposed.